deploy/openshift/templates/che-plugin-registry.yml

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2012-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: che-plugin-registry-service
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: che-plugin-registry
+    name: che-plugin-registry
+  spec:
+    replicas: 1
+    selector:
+      app: che-plugin-registry
+      deploymentconfig: che-plugin-registry
+    strategy:
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: che-plugin-registry
+          deploymentconfig: che-plugin-registry
+      spec:
+        containers:
+        - image: ${IMAGE}:${IMAGE_TAG}
+          imagePullPolicy: Always
```

Comment:
It may be useful to have an ability to configure this value, for example for testing purpose